bestie/server/testresult.go

```diff
@@ -133,14 +132,8 @@ func extractZippedFiles(stream bazelStream, zipFile string) error {
 			continue
 		}
 
-		//  Look for a matching file name using regular expression patterns.
-		match := ""
-		for _, re := range allRE {
-			match = re.FindString(filepath.Base(file.Name))
-			if len(match) > 0 {
-				break
-			}
-		}
+		//  Look for a matching file name using a regular expression pattern.
+		match := pbFileRE.FindString(filepath.Base(file.Name))
```

Comment:
...this then becomes `strings.HasSuffix(filename, ".metrics.pb")` :)

Reply:
Yeah, I was too fixated on the regex that I didn't consider the simpler approach. Will change this in my next PR, post merge of this PR.